stable/artifactory/templates/nginx-deployment.yaml

```diff
@@ -37,37 +37,18 @@ spec:
       - name: {{ .Values.imagePullSecrets }}
     {{- end }}
       initContainers:
-    {{- if .Values.nginx.persistence.enabled }}
-      - name: "remove-lost-found"
+      - name: "setup"
         image: "{{ .Values.initContainerImage }}"
         imagePullPolicy: {{ .Values.nginx.image.pullPolicy }}
         command:
         - '/bin/sh'
         - '-c'
-        - 'rm -rfv {{ .Values.nginx.persistence.mountPath }}/lost+found'
+        - >
+          [ -d {{ .Values.nginx.persistence.mountPath }}/lost+found ] && rm -rfv {{ .Values.nginx.persistence.mountPath }}/lost+found;
```

Comment:
@eldada is it really necessary to check I
if the directory exists? mew -rf will be good even when the directory doesn’t exist

Reply:
Agree. Removed.